scene/resources/audio_stream_sample.cpp

```diff
@@ -480,6 +480,10 @@ float AudioStreamSample::get_length() const {
 	return float(len) / mix_rate;
 }
 
+bool AudioStreamSample::is_polyphonic() const {
```

Comment:
I would most likely change `is_polyphonic` for `is_monophonic` because I think in practice most streams should be polyphonic unless marked as monophonic. Generally with APIs we check the exception rather than the rule to make the code easier to understand.

Reply:
I personally prefer `is_polyphonic` because the documentation makes more sense that way. "Do you support X" is more understandable than "do you explicitly NOT support x". Even though false is the exception I'd still prefer to keep the negation in the codebase rather than in the docs. I don't feel too strongly about it though. I implemented the change, so feel free to take a look at the diff for c7dad600343099c43c405d494555c46c001000ce

Let me know either way. I can squash the commits with or without that change.